src/pages/what-we-do.js

```diff
@@ -0,0 +1,48 @@
+import React from "react"
+import { graphql } from 'gatsby'
+import get from 'lodash/get'
+
+import Layout from "../components/layout"
+import SEO from "../components/seo"
+
+const WhatWeDo = (props) => {
```

Comment:
My only comment here is definitely use container components. The structure would be like this:

```
// This takes the props from gatsby
function WhatWeDoContainer(data) {
    // This takes a specific set of props that are human readable
    return <WhatWeDo title={data.title} cards={data.cards} ... />
}
```